gnomad_qc/v3/sample_qc/sample_qc.py

```diff
@@ -50,7 +89,9 @@ def compute_sample_qc() -> hl.Table:
             remove_hard_filtered_samples=False
         )
     )
-    mt = mt.filter_rows(hl.len(mt.alleles) > 1)
+
+    # Remove centromeres and telomeres incase they were included and any reference blocks
+    mt = mt.filter_rows(~hl.is_defined(telomeres_and_centromeres.ht()[mt.locus]) & (hl.len(mt.alleles) > 1))
```

Comment:
It's really two separate filters. I think the `telomeres_and_centromeres` filter would be best in the filtering because it's not really specific to sparse MTs. The ref block filter is more for the sparse MT. I put in a PR with those added to `gnomad_methods` and use them, but let me know if you think I should switch it back or have other thoughts on it.